package.json

```diff
@@ -46,6 +46,7 @@
     "lint:fix": "prettier --write '**/*.{js,jsx,ts,tsx}'"
   },
   "dependencies": {
+    "@types/react-native-video": "^5.0.9",
```

Comment:
was this meant to be on the top level package.json

Reply:
doh no :D thanks for catching